lib/blackbox/VmBlackBox.rb

```diff
@@ -217,479 +151,58 @@ def validate(settings, check_vmId = true)
         end
       end
 
-			return {:results=>{:created=>false, :exists=>self.exist?, :configured=>self.configured?, :bbName=>@bbName, :vmName=>@config_name},
-				      :config=>@cfg}
+      return {:results=>{:created=>false, :exists=>self.exist?, :bbName=>@bbName, :vmName=>@config_name},
+              :config=>@cfg}

```

Comment:
I noticed that the `:configured => self.configured?` was removed.  Is that right?  I see configured? method was removed, but is the caller expecting that value to be nil in the hash?